extensions/wikia/AdEngine/js/lookup/prebid/adapters/rubicon.js

```diff
@@ -27,12 +28,48 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.rubicon', [
 			}
 		};
 
+	function getAdContext() {
+		return adContext.getContext();
+	}
+
+	function getTargeting(slotName, skin) {
+		var provider = skin === 'oasis' ? 'gpt' : 'mobile',
+			s1 = getAdContext().targeting.wikiIsTop1000 ? adLogicZoneParams.getName() : 'not a top1k wiki';
+
+		return {
+			pos: slotName,
+			src: provider,
+			s0: adLogicZoneParams.getSite(),
+			s1: s1,
+			s2: adLogicZoneParams.getPageType(),
+			lang: adLogicZoneParams.getLanguage()
+		};
+	}
+
 	function isEnabled() {
-		return adContext.getContext().bidders.rubicon && !instartLogic.isBlocking();
+		return getAdContext().bidders.rubicon && !instartLogic.isBlocking();
 	}
 
-	function prepareAdUnit(slotName, config) {
-		var adUnit = {
+	function prepareAdUnit(slotName, config, skin) {
+		var targeting = getTargeting(slotName, skin),
+			adUnit,
+			bidParams;
+
+		bidParams = {
+			accountId: rubiconAccountId,
+			siteId: rubiconSiteId,
+			zoneId: config.zoneId,
+			name: slotName,
+			position: config.position,
+			inventory: targeting,
```

Comment:
In display we're using `config.targeting` by default. Do you think we should use it here as well?